core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -109,7 +105,6 @@
           AtomicInteger externalSize,
           OakSerializer<K> keySerializer,
           OakSerializer<V> valueSerializer, ThreadIndexCalculator threadIndexCalculator) {
-        this.memoryManager = memoryManager;
```

Comment:
Not in the chunk. ;)

Reply:
Perhaps it's better to keep a reference to it in the chunk, so that it can pass it on to handles when creating new ones. That way handles can have their own reference and shouldn't have to take a memory manager argument in some of their methods